src/pages/patientView/patientHeader/GenomeDrivenDiagnosis.tsx

```diff
@@ -0,0 +1,197 @@
+                <span style={{paddingLeft:2}}>
+                    {this.predictionContent()}
+                </span>
+            </DefaultTooltip>
+        );
+    }
+
+    @computed private get isInAllowedCancerTypes() {
+        return GDD_CANCER_TYPES.indexOf(this.cleanCancerType(this.props.prediction.Diagnosed_Cancer_Type).toLowerCase()) > -1;
+    }
+
+    @computed private get hasMatchingDiagnosis() {
+        return this.props.prediction.Pred1 && this.cleanCancerType(this.props.prediction.Pred1).toLowerCase() === this.cleanCancerType(this.props.prediction.Diagnosed_Cancer_Type).toLowerCase()
+    }
+
+    @computed private get confidenceAsPerc() {
+        return Math.round(parseFloat(this.props.prediction.Conf1) * 100)
+    }
+    
+    @computed private get isHighConfidence() {
+        return this.confidenceAsPerc > HIGH_CONFIDENCE_PREDICTION;
+    }
+
+    @computed private get isMediumConfidence() {
+        return this.confidenceAsPerc > MEDIUM_CONFIDENCE_PREDICTION && this.confidenceAsPerc <= HIGH_CONFIDENCE_PREDICTION;
+    }
+
+    private predictionContent() {
+        if (!this.isInAllowedCancerTypes || this.hasMatchingDiagnosis) {
+            return <a>(GDD)</a>
+        } else {
+            let textType = "";
+            let icon:JSX.Element|null = null;
+            if (this.isHighConfidence) {
+                textType = "text-danger";
+                icon = <i className="fa fa-exclamation-triangle" style={{color:"darkred"}}></i>;
+            } else if (this.isMediumConfidence) {
+                textType = "text-warning";
+                icon = <i className="fa fa-exclamation-triangle" style={{color:"darkorange"}}></i>;
+            }
+
+            return <a>
+                (GDD: <span className={textType} data-test="gdd-prediction">
+                    {this.cleanCancerType(this.props.prediction.Pred1)}
+                </span> {icon})
+            </a>
+        }
+    }
+
+    private tooltipContent() {
+        return (
+            <div style={{maxWidth:250}}>
+                <h6 style={{textAlign:"center"}}>
+                    Genome Driven Diagnosis (GDD)
+                </h6>
+                {   // Show warning when no diagnosis found
+                    (!this.props.prediction.Diagnosed_Cancer_Type && (
+                    <p className="small text-danger" style={{textAlign:"center"}}>
+                        No diagnosed cancer type found.
+                    </p>
+                  )) || 
+                  // Show warning when diagnosis not part of 22 types
+                  (!this.isInAllowedCancerTypes && (
+                    <p className="small text-danger" style={{textAlign:"center"}}>
+                        The diagnosed
+                        cancer type "{this.props.prediction.Diagnosed_Cancer_Type}" is not
+                        part of the 22 tumor types GDD can predict.
+                    </p>
+                  ))
+                }
+                <SimpleTable
```

Comment:
I would not use SimpleTable unless you are going to use a feature of it.  Just use a plain html table and give it a class="table"    That we get consistent style but not the probably pointless dependency